tests/net/util.rs

```diff
@@ -16,3 +18,12 @@ macro_rules! try_some {
         }
     };
 }
+
+/// Return true with a certain `probability` ([0 .. 1.0]).
+pub fn randomly(probability: f32) -> bool {
+    assert!(probability <= 1.0);
+    assert!(probability >= 0.0);
+
+    let mut rng = rand::thread_rng();
```

Comment:
This is not reproducible. Please pass an RNG as an argument and use it instead.

Reply:
Eliminated thread_rng